### PR TITLE
fix: update holesky / sepolia repository layout

### LINKED ISSUE #6821: Review repository urls of network configs
- [x] Layout of upstream network config repository is being changed https://github.com/eth-clients/holesky/pull/104. Need to review that we still pull the configs from the right path for each network.

https://github.com/ChainSafe/lodestar/blob/deafa4f162860b7677cd0b63f32eab4b47d206d4/packages/cli/src/networks/holesky.ts#L6-L7

- [x] There are also changes done in Lighthouse https://github.com/sigp/lighthouse/pull/5850, renaming `deploy_block.txt` which we use in our sim tests to configure Lighthouse.

https://github.com/ChainSafe/lodestar/blob/deafa4f162860b7677cd0b63f32eab4b47d206d4/packages/cli/test/utils/crucible/clients/validator/lighthouse.ts#L68

### PR DESCRIPTION
**Pull request by barnabasbusa**
**Motivation**

Fixes repository layout changes according to: 
* https://github.com/eth-clients/holesky/pull/104
* https://github.com/eth-clients/sepolia/pull/78

Might fully close #6821

### CODE REVIEW

**Review by ensi321 — DISMISSED** [review]
lgtm after fixing the lint issue

File: packages/config/src/chainConfig/networks/sepolia.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import {ChainConfig} from "../types.js";
 import {chainConfig as mainnet} from "../configs/mainnet.js";
 
 // Sepolia beacon chain config:
-// https://github.com/eth-clients/sepolia/blob/main/bepolia/config.yaml
+// https://github.com/eth-clients/sepolia/blob/main/metadata/config.yaml
```
Comment (nflaig) [inline packages/config/src/chainConfig/networks/sepolia.ts:7]: We are still pulling sepolia network config from the merge-testnets repo
https://github.com/ChainSafe/lodestar/blob/8b6ecc44c1892115d0e7396b2a5f0ae8fc0d9e6f/packages/cli/src/networks/sepolia.ts#L5-L6

is this something we wanna update in this PR as well?

**Review by nflaig — COMMENTED** [review]
Let's wait with merging this until upstream PRs are merged

File: packages/config/src/chainConfig/networks/sepolia.ts (hunk 1 above)
Comment (barnabasbusa) [reply packages/config/src/chainConfig/networks/sepolia.ts:7]: yes please. The merge-testnets repo has been archived now.

**Comment by barnabasbusa** [comment]
Its probably time to also get rid of all ropsten/goerli code as well.

**Comment by nflaig** [comment]
> Its probably time to also get rid of all ropsten/goerli code as well.

already done, your branch is not up to date
- see https://github.com/ChainSafe/lodestar/pull/6870

**Review by nflaig — DISMISSED** [review]

**Review by nflaig — APPROVED** [review]

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.20.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.20.0) :tada:

**Pull request merged**